src/EnergyPlus/FileSystem.cc

```diff
@@ -98,9 +101,13 @@ namespace FileSystem {
         // filename. Do we really need that though?
         // path.make_preferred();
         fs::path result = path;
+#ifdef _WIN32
+        result.make_preferred();
+#else
```

Comment:
in makeNativePath, avoid going to a string() if not needed (and will potentially crash)